packages/alfa-rules/src/sia-r1/rule.ts

```diff
@@ -51,10 +51,10 @@ function hasDocumentElement(document: Document): boolean {
   return false;
 }
 
-function isTitle(element: Element, context: Node): boolean {
-  if (getElementNamespace(element, context) !== Namespace.HTML) {
-    return false;
-  }
-
-  return element.localName === "title";
+function isTitle(node: Node, context: Node): node is Element {
+  return new ElementChecker()
+    .withName("title")
+    .withContext(context)
+    .withNamespace(Namespace.HTML)
+    .evaluate(node) as boolean;
```

Comment:
This is typically the cast I'd like to avoid. The toplevel checker never return browser specific values…